packages/create/README.md

```diff
@@ -24,6 +24,15 @@ Calling `sap-ux add` allows adding a feature to a project.
 ## remove
 Calling `sap-ux remove` allows removing a feature to a project.
 
+## adp-add
```

Comment:
I would prefer adding it to `add` and not have the additional `adp-add`.

Reply:
And then for the full command `add data-source-change`.

Reply:
Fixed with: 8fbf2b395b9db4e830c8f261b6eefcac83f56f4a

Reply:
I didn't notice the first time that you prefered shuffling the words in the command. Could you give me some more details on the convention in such cases, as I couldn't notice any similar command already existing?
Also, I will update the readme.

Reply:
Sure, the idea was to have commands consisting of two parts (1) activity and (2) object. As activities, we have so far `add`, `remove` and `generate` (`eject` likely coming soon). The object should describe what is being modified. In your case you add a new flex change that changes the data source of an app, so I would say a "datasource change" therefore, the suggested command `add data-source-change`.
I would expect that we soon also get more things like `*-change` objects for other flex changes

Reply:
The other manifest change wizards are as follows:
- add-annotations-to-odata 
- add-component-usages
- add-new-model
- change-inbound

For example, in the change file resulting from add-annotations-to-odata wizard, the ChangeType property will be appdescr_app_addAnnotationsToOData. How do you think we could fit to the logic you described?

Reply:
Hmm ... good question, I'd go for descriptive concise names since the user is supposed to enter them in the cli.

First shot:
* `add data-source-change`
* `add new-model-change` (btw, when is the difference between that one and the data source?)
* `add inbound-nav-change`
* `add component-usage-change`
* `add local-annotations` (I removed change here because that one does more than just adding a change)

Quite frankly, they are not concise or intuitive, maybe we should "play" with the verb and introduce "change" and have something like:
* `change data-source`
* `add new-model`
* `change inbound-navigation`
* `add component-usage`
* `add local-annotations`